# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1264026263 18000
#      Wed Jan 20 17:24:23 2010 -0500
# Node ID 2c28f9d0360f3081e2a0263148159a2bd7287b3a
# Parent  b6b65e71049b0f158f78d174172bb1e0a7e07f46
src/TEMPLATE-INST/*.cc: remove Emacs local variable settings

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2010-01-20  John W. Eaton  <jwe@octave.org>
+
+	* TEMPLATE-INST/Array-os.cc, TEMPLATE-INST/Array-tc.cc,
+	TEMPLATE-INST/Array-sym.cc: Delete Emacs local-variable settings.
+
 2010-01-20  Jaroslav Hajek  <highegg@gmail.com>
 
 	* xpow.cc (xpow, elem_xpow): Don't do redundant real->complex
 	conversions.
 
 2010-01-20  Jaroslav Hajek  <highegg@gmail.com>
 
 	* data.cc: Substitute OCTAVE_QUIT -> octave_quit ().
diff --git a/src/TEMPLATE-INST/Array-os.cc b/src/TEMPLATE-INST/Array-os.cc
--- a/src/TEMPLATE-INST/Array-os.cc
+++ b/src/TEMPLATE-INST/Array-os.cc
@@ -40,14 +40,8 @@ typedef printf_format_elt* printf_format
 NO_INSTANTIATE_ARRAY_SORT (scanf_format_elt_ptr);
 INSTANTIATE_ARRAY (scanf_format_elt_ptr, OCTINTERP_API);
 
 NO_INSTANTIATE_ARRAY_SORT (printf_format_elt_ptr);
 INSTANTIATE_ARRAY (printf_format_elt_ptr, OCTINTERP_API);
 
 NO_INSTANTIATE_ARRAY_SORT (octave_stream);
 INSTANTIATE_ARRAY (octave_stream, OCTINTERP_API);
-
-/*
-;;; Local Variables: ***
-;;; mode: C++ ***
-;;; End: ***
-*/
diff --git a/src/TEMPLATE-INST/Array-sym.cc b/src/TEMPLATE-INST/Array-sym.cc
--- a/src/TEMPLATE-INST/Array-sym.cc
+++ b/src/TEMPLATE-INST/Array-sym.cc
@@ -32,14 +32,8 @@ along with Octave; see the file COPYING.
 #include "oct-obj.h"
 #include "symtab.h"
 
 typedef symbol_record* symbol_record_ptr;
 
 NO_INSTANTIATE_ARRAY_SORT (symbol_record_ptr);
 
 INSTANTIATE_ARRAY (symbol_record_ptr, OCTINTERP_API);
-
-/*
-;;; Local Variables: ***
-;;; mode: C++ ***
-;;; End: ***
-*/
diff --git a/src/TEMPLATE-INST/Array-tc.cc b/src/TEMPLATE-INST/Array-tc.cc
--- a/src/TEMPLATE-INST/Array-tc.cc
+++ b/src/TEMPLATE-INST/Array-tc.cc
@@ -36,14 +36,8 @@ along with Octave; see the file COPYING.
 
 #include "oct-sort.cc"
 
 NO_INSTANTIATE_ARRAY_SORT (octave_value);
 
 INSTANTIATE_ARRAY (octave_value, OCTINTERP_API);
 
 template class OCTINTERP_API Array2<octave_value>;
-
-/*
-;;; Local Variables: ***
-;;; mode: C++ ***
-;;; End: ***
-*/
